docs/deployment/kubernetes-deployment.md

```diff
@@ -0,0 +1,384 @@
+---
+title: Deploy .NET Aspire projects to Kubernetes
+description: Learn how to deploy .NET Aspire projects to Kubernetes clusters using Aspir8 or manual YAML generation.
+ms.date: 01/16/2025
+ms.topic: how-to
+---
+
+# Deploy a .NET Aspire project to Kubernetes
+
+.NET Aspire projects are designed to run in containerized environments. Kubernetes is a popular container orchestration platform that can run .NET Aspire projects in any cloud provider or on-premises environment. This article walks you through deploying .NET Aspire solutions to Kubernetes clusters using different approaches. You'll learn how to complete the following tasks:
+
+> [!div class="checklist"]
+>
+> - Deploy using the Aspir8 tool for automated manifest generation
+> - Manually create and customize Kubernetes YAML manifests
+> - Use Visual Studio Code with Kubernetes extensions for development workflow
+> - Configure .NET Aspire for Kubernetes deployment
+
+[!INCLUDE [aspire-prereqs](../includes/aspire-prereqs.md)]
+
+## Prerequisites
+
+Before deploying to Kubernetes, ensure you have:
+
+- A running Kubernetes cluster. You can use:
+  - [Azure Kubernetes Service (AKS)](/azure/aks/learn/quick-kubernetes-deploy-portal) for cloud deployment
+  - [Docker Desktop](https://docs.docker.com/desktop/kubernetes/) for local development
+  - [minikube](https://minikube.sigs.k8s.io/docs/start/) for local testing
+  - Any other Kubernetes distribution
+- [kubectl](https://kubernetes.io/docs/tasks/tools/) configured to access your cluster
+- [Docker](https://docs.docker.com/get-docker/) for building container images
+
+> [!TIP]
+> For production deployments on Azure, see [Quickstart: Deploy an Azure Kubernetes Service (AKS) cluster using the Azure portal](/azure/aks/learn/quick-kubernetes-deploy-portal).
```

Comment:
You now have two identical headings, let's address this please. Here's what it looks like as rendered with the include and the other section:

![image](https://github.com/user-attachments/assets/45fb386e-7d2e-4eda-b0e1-41f0c7a299ba)